### ISSUE TITLE
Loss term

### ISSUE THREAD

**Issue by bobwan1995** [Flaick/GroundR#2]
I found you used region-level supervision in train_unsupervised_withval.py line 142,  i.e., loss_att term. In the unsupervised setting, we can't get access to this supervision, right?

**Comment by Flaick** [comment]
Yes, you are right, I haven't tested it in a fully unsupervised manner. But you can set the weight 'L' as 0 for loss_att term.
